docs/readmes/basics/prerequisites.md

```diff
@@ -15,27 +15,32 @@ Install the following tools:
 1. [Docker](https://www.docker.com) and Docker Compose
 2. [Homebrew](https://brew.sh/) *only* for MacOS users
 3. [VirtualBox](https://www.virtualbox.org/)
-3. [Vagrant](https://vagrantup.com)
+4. [Vagrant](https://vagrantup.com)
 
 Replace `brew` with your OS-appropriate package manager as necessary, or see
 the [pyenv installation instructions](https://github.com/pyenv/pyenv#installation).
 
 ```bash
+brew install go@1.13
 brew install pyenv
 
-# Replace .zshrc with your appropriate shell RC file
+# Replace .zshrc with your appropriate shell RC files
```

Comment:
Nit: I like this as singular

Reply:
Yeah my bad, I thought it was initially a typo(It was spelled "fiels") when I read it so I wanted to fix it. Looking back, I most likely messed up the spelling myself and tried to correct it without double checking. The original version is definitely right.